kernelci/runtime/lava/__init__.py

```diff
@@ -49,6 +49,9 @@ def submit(self, job_path):
     def job_file_name(self, params):
         return '.'.join([params['name'], 'yaml'])
 
+    def get_job_id(self, job_object):
+        return job_object
```

Comment:
Is the `job_object` just an id in this case?

Reply:
Yes